.github/workflows/automate.yaml

```
@@ -2,7 +2,8 @@ name: Daily Data Retrieval Automation
 
 on:
   schedule:
-    - cron: "0 0 */2 * *" # Runs every other day at 12AM
+    # TODO - Fix this time issue
```

Create a new issue as to not forget to follow up on this in the future.
```suggestion
    # TODO(#<ISSUE NUMBER>): Fix this time issue.
```

---

Resolved [here](https://github.com/acmcsufoss/acmOpportunities/pull/7/commits/e873aa82ae04037e5f55d8141c2e11e7f4be4012)